DeepCat.js

```diff
@@ -202,17 +198,26 @@
 	}
 
 	function getSearchTerms( input ) {
-		return input.match( searchTermRegExp( keyString ) );
+		return input.match( '(-?\\b' + keyString + '\\s*'
+							+ '(?:("([^\\"]|\\.)+")' 			// search for keyword:"term including \"escaped\" quotes"
+							+ '|(?!' + keyString + ')\\S+))' 	// search for keyword:term excluding keyword term as parameter
```

Comment:
I wonder what this is supposed to do or to prevent? At the moment this line does not do anything because the line below will match anyway what this line skipped.